src/mame/ibm/thinkpad600.cpp

```diff
@@ -0,0 +1,89 @@
+// license:BSD-3-Clause
+// copyright-holders:
+/*************************************************************************************************************
+Skeleton driver for IBM ThinkPad 600 series.
+The IBM ThinkPad 600 series was a series of notebook computers introduced in 1998 by IBM as an lighter and 
+slimmer alternative to the 770 series. Three models were produced, the 600, 600E, and 600X.
+
+Hardware for the 600E model.
+  Main PCB:
+    -Intel Pentium II 366 Mobile MMC-2 (PMG36602002AA).
+    -Texas Instruments PCIbus SN104698GFN.
+    -Intel PCIset FW82371EB (PCI-TO-ISA / IDE XCELERATOR PIIX4)
```

Comment:
MMC-2 should be i440bx-ish in a chip. Is it possible to run a PCI scan on this? 
FreeDOS `pcisleep L`, or [pci.exe](https://www.os2world.com/wiki/index.php/PCI.EXE:_A_powerful_sniffing_utility) for DOS, or `lspci -vxxx` on *nix.

Reply:
No, sorry, it won't work anymore.
Anyway, it's for sure a PMG36602002AA CPU, TI SN104698GFN and Intel FW82371EB, see the pics:

![88a36665-f814-410a-b3b8-36a873dfa849](https://github.com/user-attachments/assets/4df7306e-94f9-4146-b8ac-0f3b7c8d9b87)

![IBM Thinkpad 600E 27L5053 Mainboard BOTTOM](https://github.com/user-attachments/assets/4088046d-bdca-4138-b637-855a8d00712c)